compiler/testSuite/10-intersect/construct1-v.bal

```diff
@@ -2,6 +2,6 @@ type R1 record {| int? x; |};
 type R2 record {| int|string x; |};
 
 public function main() {
-    R1&R2 r = { x: 1 }; // @error
+    R1&R2 r = { x: 1 };
```

Comment:
Need a comment here, saying we are deviating from the spec (and similarly in other text case other than 14-intersection)

Reply:
Fixed